install/install.sh

```diff
@@ -112,5 +112,5 @@ then
   echo "Installation complete."
 else
   echo "Installation failed. Review the install logs."
-  exit $?
+  exit 1
```

Comment:
The `$?` will always be 0 because it will be the exit code of the `echo` command above. This will make the script fail whenever a failure is raised when running ansible-playbook.